rke/k8s_version_info.go

```diff
@@ -117,8 +131,15 @@ func loadK8sVersionInfo() map[string]v3.K8sVersionInfo {
 		},
 		// The Calico/Canal template in this version use functions that are only available in RKE v1.0.0 and up
 		// This version includes nodelocal dns only available in RKE v1.0.7 and up
+		// It also includes ingress-nginx 0.32.0
+		"v1.17.5-rancher1-2": {
+			MinRancherVersion: "2.3.7-rc0",
```

Comment:
Missed this earlier and it's a nitpick really cause it only affects 2.3.7-rcs. But this should be 2.3.7 if we want to keep the same min convention of how we introduced v1.15.12-rancher1-2

Reply:
removed `-rc0`